lib/src/widgets/buttons/ds_play_button_rounded.widget.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import '../../enums/ds_align.enum.dart';
+import '../../themes/colors/ds_colors.theme.dart';
+import '../../utils/ds_utils.util.dart';
+
+class DSPlayButtonRounded extends InkWell {
+  DSPlayButtonRounded({
```

Comment:
Following the English rules, I think that the right name would be DSRoundedPlayButton.

Reply:
Done